rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import (
+    Column,
+    Enum,
+    ForeignKey,
+    Integer,
+    String
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    rounds_limit = Column(Integer)
+    winner_id = Column(Integer, ForeignKey('user.id'), nullable=False)
+
+    player_1 = relationship('User', foreign_keys=[player_1_id])
+    player_2 = relationship('User', foreign_keys=[player_2_id])
+    winner = relationship('User', foreign_keys=[winner_id])
+
+    rounds_remaining = relationship('Round', foreign_keys='Round.id')
```

Comment:
I think this should be an attribute using `@property` that returns an integer representing the _number_ of rounds which are incomplete, based on `rounds_limit`. Basically:
```python
@property
def rounds_remaining(self):
    rounds_complete = 0

    for round in self.rounds:
        if round.winning_choice is not None:
            rounds_complete += 1

    return self.rounds_limit - rounds_complete
```

Note that we'll also need a `rounds` relationship based on the `Round.game_id` FK.